libs/vkd3d/vkd3d_private.h

```diff
@@ -1728,7 +1739,7 @@ struct d3d12_command_list
 
     bool predicate_enabled;
     VkDeviceAddress predicate_va;
-
+        
```

Comment:
Weird space

Reply:
this is weird :) actually there is a space in original file after `VkDeviceAddress predicate_va;`